### DISCUSSION TITLE
Colours or room backgrounds to easily visualise different rooms

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#780] [Product Feedback]
[Not Product approved, this request came via a conversation with a user]

Setting a wallpaper for a room based on it's encryption status or bridged status - set by the room admin - so that it can be easily and quickly identified as a room that should not contain sensitive information. 

For example; A room that bridges to WhatsApp share information outside of Element in order to communicate with the third party app. However, these rooms look the same in Element and there's no visual differentiation or reminder in the timeline. 

👀  @Deadwormjumping

**Comment by Deadwormjumping** [comment]
As a user i would want the ability to set the background of a timeline to an image or solid colour.  This becomes more desired in certain situations like creating bridged rooms to Whatsapp for instance where a greater visual cue would help to indicate a bridged room.  

This would also apply to a user who likes the novelty of defining the background to a room or 1:1 chat.

**Reply by daniellekirkwood** [reply]
We would need to discuss who and how chooses the background. 
In WhatsApp for eg each user's setting is personal and only applies to them. We could use the same approach - or have some presets that enables colours based on room type (encrpyted, bots/bridges, etc.) to introduce consistency
